### PR TITLE
add mozdata schemas

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: schemas/mozdata/sql/sql.1.parquetmr.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+message sql {
```
[the remark this answers is not in the document]
Comment (relud) [reply schemas/mozdata/sql/sql.1.parquetmr.txt]: this is that

File: schemas/mozdata/readTable/readTable.1.parquetmr.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+message readTable {
+  required binary apiVersion (UTF8);
+  optional binary detectedUri (UTF8);
+  optional binary detectedVersion (UTF8);
```
[the remark this answers is not in the document]
Comment (relud) [reply schemas/mozdata/readTable/readTable.1.parquetmr.txt]: yes, they are

File: schemas/mozdata/listRDDs/listRDDs.1.parquetmr.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+message listRDDs {
+  required binary apiVersion (UTF8);
+  required binary sourcesJson (UTF8);
+  required binary metadata_documentId (UTF8);
+  optional binary metadata_geoCountry (UTF8);
```
[the remark this answers is not in the document]
Comment (relud) [reply schemas/mozdata/listRDDs/listRDDs.1.parquetmr.txt]: good point, we don't actually care. i'll drop these

File: schemas/mozdata/readRDD/readRDD.1.parquetmr.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+message readRDD {
+  required binary apiVersion (UTF8);
```
[the remark this answers is not in the document]
Comment (relud) [reply schemas/mozdata/readRDD/readRDD.1.parquetmr.txt]: added

**Comment by relud** [comment]
updated to use templating.

> consolidating these into a single table with a field to differentiate them (and make `partition` optional)

hmmm. @fbertsch preference between single-table and table-per-api ?

**Comment by fbertsch** [comment]
@relud I would prefer a single data table, and camel-cased docTypes (since that conforms with existing docTypes).

File: templates/include/mozdata/base.1.schema.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+"apiVersion": {
```
Comment (fbertsch) [inline templates/include/mozdata/base.1.schema.json]: Is API version the only thing we want to record here (I think we had this discussion in the MozData PR as well). I could imagine: Spark version, Scala version, Metastore URL. That may be it off the top of my head.

Comment (fbertsch) [reply templates/include/mozdata/base.1.schema.json]: I suppose? It seems odd to me that we have to get data review for internal tools.

**Pull request merged**